osbuild/formats/v2.py

```diff
@@ -270,17 +270,26 @@ def load(description: Dict, index: Index) -> Manifest:
     pipelines = description.get("pipelines", [])
 
     manifest = Manifest()
+    source_ids = set()
```

Comment:
Maybe `source_refs` because in the osbuild terminology they are called "references"?